build.gradle

```diff
@@ -24,6 +24,7 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven { url 'http://developer.huawei.com/repo/' }
```

Comment:
This line already exist in oaid library gradle

Reply:
Can you point me in where? I dont see it